WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -1,155 +1,157 @@
-#import "WPTableViewControllerSubclass.h"
 #import "CommentsViewController.h"
-#import "NewCommentsTableViewCell.h"
+#import "CommentsTableViewCell.h"
 #import "CommentViewController.h"
-#import "WordPressAppDelegate.h"
-#import "ReachabilityUtils.h"
-#import "UIColor+Helpers.h"
-#import "WPTableViewSectionHeaderView.h"
+#import "CommentService.h"
 #import "Comment.h"
+
+#import "WordPress-Swift.h"
+#import "WPTableViewHandler.h"
+#import "WPGUIConstants.h"
+#import "WPNoResultsView.h"
+#import "UIView+Subviews.h"
 #import "ContextManager.h"
-#import "CommentService.h"
 
-@interface CommentsViewController ()
 
-@property (nonatomic,strong) NSIndexPath *currentIndexPath;
-@property (nonatomic,assign) BOOL moreCommentsAvailable;
+CGFloat const CommentsStandardOffset        = 16.0;
+CGFloat const CommentsSectionHeaderHeight   = 24.0;
+CGRect const CommentsActivityFooterFrame    = {0.0f, 0.0f, 30.0f, 30.0f};
+CGFloat const CommentsActivityFooterHeight  = 50.0f;
+
+
+@interface CommentsViewController () <WPTableViewHandlerDelegate, WPContentSyncHelperDelegate>
+@property (nonatomic, strong) WPTableViewHandler        *tableViewHandler;
+@property (nonatomic, strong) WPContentSyncHelper       *syncHelper;
+@property (nonatomic, strong) WPNoResultsView           *noResultsView;
+@property (nonatomic, strong) UIActivityIndicatorView   *footerActivityIndicator;
+@property (nonatomic, strong) UIView                    *footerView;
 @end
 
-@implementation CommentsViewController
 
-CGFloat const CommentsStandardOffset = 16.0;
-CGFloat const CommentsSectionHeaderHeight = 24.0;
+@implementation CommentsViewController
 
 - (void)dealloc
 {
-    DDLogMethod();
     [[NSNotificationCenter defaultCenter] removeObserver:self];
-}
-
-- (NSString *)noResultsTitleText
-{
-    return NSLocalizedString(@"No comments yet", @"Displayed when the user pulls up the comments view and they have no comments");
+    self.syncHelper.delegate = nil;
+    self.tableViewHandler.delegate = nil;
 }
 
 - (void)viewDidLoad
 {
-    DDLogMethod();
-
+    NSParameterAssert(self.view);
+    NSParameterAssert(self.tableView);
```

Comment:
I've initially added a *nib* for this ViewController, but at some point i ended up ditching that. You're right, since those two outlets are created by *UITableViewController* itself, i'll go ahead and nuke the asserts!